x86/bell/dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:6.11-alpine
+
+LABEL MAINTAINER "Dashlorde <zhouyunlu0216@gmail.com>"
+LABEL MAINTAINER "Abdurrachman Mappuji <mappuji@ole.org>"
+LABEL MAINTAINER "dogi <stefan@unterhauser.name>"
+
+ENV version '0.13.21'
+
+RUN mkdir -p /app/server/src
```

Comment:
No need to do `mkdir -p /app/server/src` since it will create new layer. Just do the `WORKDIR` here.